storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -242,6 +242,19 @@ xb_fil_cur_open(
 
 	/* Allocate read buffer */
 	cursor->buf_size = XB_FIL_CUR_PAGES * page_size;
+	if (opt_read_buffer_size != 0) {
+		ulint buf_size = cursor->buf_size;
+		const char *factors[] = {"b", "Kb", "Mb", NULL};
```

Comment:
Sure, but suffices here are not for parsing option values, but for reporting custom buffer size.

The whole point of having default value of `0` was to be able distinguish user set value from default one. In my opinion, having specific value set as default value would complicate things a bit:
* that would create unnecessary dependency of option-specifying code to the engine-stuff
* default value may change in future engine versions (or different engines may have different default values)
* it is hard to tell if specific value was explicitly set by the user or was that a default one.